pkg/k8s/connect.go

```diff
@@ -1,12 +1,28 @@
 package k8s
 
 import (
+	keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
 	keptnutils "github.com/keptn/kubernetes-utils/pkg"
 	"k8s.io/client-go/kubernetes"
+	"keptn-sandbox/job-executor-service/pkg/config"
 )
 
+// K8s is used to interact with kubernetes jobs
+type K8s struct {
+}
+
+//go:generate mockgen -source=connect.go -destination=fake/connect_mock.go -package=fake Interface
+
+// Interface is used to interact with kubernetes jobs
+type Interface interface {
```

Comment:
Can you name this to indicate that it groups methods for kubernetes? `Interface` is a bit generic. :D

Reply:
Maybe name the interface `K8s` and the struct `k8sImpl`

Reply:
Renamed:
* package `k8s` to `k8sutils`
* `Interface` to `K8s`
* struct `K8s` to `K8sImpl`

Reply:
Also made k8sImpl non-exported and added a function `NewK8s` that creates and returns an k8sImpl